Research/Research/RSDTaskResultObject.swift

```diff
@@ -32,6 +32,7 @@
 //
 
 import Foundation
+import JsonModel
```

Comment:
Just curious on the thought behind making JsonModel its own framework?

Reply:
I wanted to be explicit about this being a reusable "framework" that extends the functionality of the Codable protocol to allow for polymorphism and documentation. It does not explicitly reference any of *our* (older) model for handling these things and can therefore be used for other projects.